administrator/language/de-DE/de-DE.plg_editors_tinymce.ini

```diff
@@ -142,12 +142,12 @@ PLG_TINY_FIELD_VISUALBLOCKS_LABEL="Visuelle Blöcke"
 PLG_TINY_FIELD_WORDCOUNT_DESC="Wortzähler an/aus"
 PLG_TINY_FIELD_WORDCOUNT_LABEL="Wortzähler"
 PLG_TINY_LEGACY_WARNING="Das <a href=\"%s\">TinyMCE-Editor-Plugin</a> wurde aktualisiert. Im Moment verwendet es eine Vorgänger-Konfiguration. Durch das Bearbeiten des Plugins können nun bestimmte Layouts verschiedenen Benutzergruppen zugeordnet werden.<br /><br /><span style=\"text-decoration: underline; color: red;\"><strong>Warnung:</strong></span><br />Bei der Bearbeitung des Plugins gehen alle vorherigen Einstellungen verloren!"
-PLG_TINY_SET_TARGET_PANEL_DESCRIPTION="<strong>Existing sets of the TinyMCE panel, and options for each.</strong><br />In each set you can add new menus or buttons from <strong>the list of available menus and buttons</strong> or remove unneeded."
+PLG_TINY_SET_TARGET_PANEL_DESCRIPTION="<strong>Vorhandene Voreinstellungen des TinyMCE Panel und Optionen für diese.</strong><br />In jeder Voreinstellung können neue Menüs und Schaltflächen aus <strong>der Liste der verfügbaren Menüs und Schaltflächen</strong> hinzugefügt oder entfernt werden."
```

Comment:
Vorschläge:
```
- <strong>Vorhandene Voreinstellungen des TinyMCE Panel und Optionen für diese.
+ <strong>Vorhandene Voreinstellungen des TinyMCE Panel und Optionen.
-<br />In jeder Voreinstellung können neue Menüs und Schaltflächen aus <strong>der Liste der verfügbaren Menüs und Schaltflächen</strong> hinzugefügt oder entfernt **werden."
+<br />In jeder Voreinstellung können Menüs und Schaltflächen aus <strong>der Liste der verfügbaren Menüs und Schaltflächen</strong> hinzugefügt oder entfernt werden.```
```
Full:
```<strong>Vorhandene Voreinstellungen des TinyMCE Panel und Optionen</strong><br />In jeder Voreinstellung können Menüs und Schaltflächen aus <strong>der Liste der verfügbaren Menüs und Schaltflächen</strong> hinzugefügt oder entfernt werden.```